pkg/podgrouper/podgrouper/plugins/defaultgrouper/default_grouper.go

```diff
@@ -176,6 +178,24 @@ func (dg *DefaultGrouper) CalcPodGroupPriorityClass(topOwner *unstructured.Unstr
 	return defaultPriorityClassForJob
 }
 
+func (dg *DefaultGrouper) calcPodGroupPreemptibility(topOwner *unstructured.Unstructured, pod *v1.Pod) v2alpha2.Preemptibility {
+	if preemptibility, found := topOwner.GetLabels()[constants.PreemptibilityLabelKey]; found {
+		if preemptibility, err := v2alpha2.ParsePreemptibility(preemptibility); err == nil {
+			return preemptibility
+		} else {
+			logger.Error(err, "Invalid preemptibility label found on top owner", "topOwner", topOwner.GetName())
+		}
+	} else if preemptibility, found = pod.GetLabels()[constants.PreemptibilityLabelKey]; found {
```

Comment:
Why lookup twice in the map?

Reply:
I'm can't find the other place we lookup in the map, can you please help me with a reference?

Reply:
Sorry, didn't realize that one was for pod and one for topOwner